### ISSUE TITLE
How do you get affine_trans in the code?

### ISSUE THREAD

**Issue by Dragon2938734** [sail-sg/mvp#24]
How do you get affine_trans in the code?

for exameple: affine_trans = torch.stack([m['affine_trans'] for m in meta], dim=1)
